website/client/src/components/groups/membersModal.vue

```diff
@@ -10,7 +10,6 @@
       id="members-modal"
       :title="$t('createGuild')"
       size="md"
-      :hide-footer="true"
```

Comment:
why was this removed?

Reply:
because i change class="modal-footer" to slot. And it work's fine
with class:
<img width="506" alt="Снимок экрана 2020-01-19 в 15 48 19" src="https://user-images.githubusercontent.com/26181595/72681380-020b6380-3ad4-11ea-97f6-0e333c553216.png">
with slot:
<img width="504" alt="Снимок экрана 2020-01-19 в 15 54 10" src="https://user-images.githubusercontent.com/26181595/72681382-06d01780-3ad4-11ea-8b62-38977ee167cd.png">

Reply:
thanks, that makes sense!